Portfolio: mobile version - middle sections

lmunozm1702 opened the pull request:
Add the 2 middle sections:
1. The "Work" section: 
- Add the 4 cards for the projects using GRID to set the position of the card list and FLEX to position the in-card data.
- The background borders and color use the corner radius as asked in the Figma template.
- The button interaction for "hover" and "activate" is developed.

2. The "About Myself" section:
- Add "About Myself" info, including social media and "My resume" links. 
- includes 3 lists for languages, frameworks, and skills experience.
(The icons for frameworks and skills were not available in Figma so I repeated the ones provided for languages.)

Add images into the /assets/images folder to rich the content.

lmunozm1702 committed:
Work section layout definition

Update index.html and style.css to define the layout of work section and work card.

Change background-image from <main> to .main-content because the background isn't present into the work section.

lmunozm1702 committed:
Add Tonic project card

Update index.html and styles.css to include the 1st info card, for "Tonic" project, into the middle section.

Add project-tonic,jpg into /assets/images project folder.

lmunozm1702 committed:
Add Multi-Post project card

Update index.html to include the 2nd. info card, for "Multi-Post Stories" project, into the middle section.

Add project-multi-post.jpg into /assets/images project folder.

lmunozm1702 committed:
Add Tonic-2 Project card

Update index.html to include the 3rd. info card, for second "Tonic" project, into the middle section.

Add project-tonic-2.png file into /assets/images folder.

Correct the ALT property for the image in 2nd card.

lmunozm1702 committed:
Add Multi-Post-2 project card

Update index.html to include the 4th. info card, for second "Multi-Post Stories" project, into the middle section.

Add project-multi-post-2.png into /assets/images project folder.

lmunozm1702 committed:
Add button interactions

Update style.css to include hover and active interaction to button element.

lmunozm1702 committed:
Add About myself section

Update index.html and style.css to include the "About Myself" section .

lmunozm1702 committed:
Add experiences lists

Update index.html and styles.css to add the experience data into the About Myself section.

Includes modifications to solve background issues into the "main-content" section.

Add images into the /assets/images folder:
- css.png
- html5.png
javascript.png

lmunozm1702 committed:
Solve background issues

Update styles.css to solve problems with the shadow and radious background on main-content and about-myself sections.

lmunozm1702 committed:
Linter STYLEINT issues

Update styles.css to solve issues:

assets/css/style.css
   7:1   ✖  Expected empty line before rule                     rule-empty-line-before
  14:12  ✖  Unexpected unit                                     length-zero-no-unit
  22:21  ✖  Expected "#ffffff" to be "#fff"                     color-hex-length
  40:21  ✖  Expected "#ffffff" to be "#fff"                     color-hex-length
 135:12  ✖  Unexpected unit                                     length-zero-no-unit
 228:15  ✖  Expected "#ffffff" to be "#fff"                     color-hex-length
 233:19  ✖  Unexpected unit                                     length-zero-no-unit
 236:21  ✖  Expected "#ffffff" to be "#fff"                     color-hex-length
 242:19  ✖  Unexpected unit                                     length-zero-no-unit
 243:17  ✖  Unexpected unit                                     length-zero-no-unit
 267:10  ✖  Expected "#000000" to be "#000"                     color-hex-length
 304:3   ✖  Unexpected shorthand "margin" after "margin-top"    declaration-block-no-shorthand-property-overrides
 304:3   ✖  Unexpected shorthand "margin" after "margin-left"   declaration-block-no-shorthand-property-overrides
 316:16  ✖  Unexpected unit                                     length-zero-no-unit
 318:10  ✖  Expected "#ffffff" to be "#fff"                     color-hex-length
 324:10  ✖  Expected "#ffffff" to be "#fff"                     color-hex-length

franclobo reviewed (commented):
Hi @lmunozm1702, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a professional README file. Please 🙏🏽  follow the instructions from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md). Kindly add a meaningful title to your pull request.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

lmunozm1702 commented:
Hi @franclobo, please your help to clarify the problem with the README.md file. The file is in the project and has passed 2 previous code reviews without comment, one yesterday and one today. Do you need me to create a new README.md for this pull request?

Regarding the title of the pull request, in the previous pull request for this project made this morning, the reviewer sent me the title and description as a Top Highlight, so I continued along the same lines, do you need me to change it too?, the Previous reviewer sent me this comment:
---------
Top highlight 🎯

The Pull Request is following the GitHub Flow ✅
The PR title and description are professional and descriptive ✅
Liners are working properly ✅
---------
Please help me, I'm very confused as I followed the same path as yesterday and today in the morning when I had no qualms with these issues.

P.S.: I don't know why some text are in bold font and another in normal font.

franclobo commented:
> Hi @franclobo, please help to clarify the problem with the README.md file. The file is in the project and has passed 2 previous code reviews without comment, one yesterday and one today. Do you need me to create a new README.md for this pull request?
> 
> Regarding the title of the pull request, in the previous pull request for this project made this morning, the reviewer sent me the title and description as a Top Highlight, so I continued along the same lines, do you need me to change it too?, the Previous reviewer sent me this comment:
> ---------
> Top highlight 🎯
> 
> The Pull Request is following the GitHub Flow ✅
> The PR title and description are professional and descriptive ✅
> Liners are working properly ✅
> ---------
> Please help me, I'm very confused as I followed the same path as yesterday and today in the morning when I had no qualms with these issues.

Hi, @lmunozm1702 please 🙏  edit your README file according to the README template of Microverse. I'm sorry whether the previous reviewers did not request this changes. About the PR edit it giving a meaningful title and describe the actions in your branch, just an example:
## Portfolio, mobile version.
- Add a header with a logo and hamburger button.
- Add a title...
- Add icons...
-....

lmunozm1702 committed:
Code Review: Update README.md

Update the README.md file to follow the Microverse template available for it.

lmunozm1702 commented:
Hi @franclobo, all done, thanks a lot for your explanations.

HAMMAS-SALEEM commented on assets/css/style.css:
```diff
@@ -1,23 +1,25 @@
-main {
-  background-image: url(/assets/images/header-shapes-mobile.png);
+body {
+  margin: 0;
+  padding: 0;
+  max-width: 375px;
+  background-color: #f7f7f9;
 }
```
- [x] Kindly remove the **max-width** to make your project expandable to the whole screen. I know responsiveness is not a part of this project but your project should be responsive on all mobile screens :blush:

lmunozm1702 replied:
It's Done!

HAMMAS-SALEEM commented on index.html:
```diff
@@ -14,41 +14,251 @@
+                </footer>
+            </article>
+        </section>
+        <section class="about-myself">
+            <h1>About Myself</h1>
+            <p class="main-normal-text">Hello I’m a software developer! I can help you build a product , feature or
```
- [x] Please replace this `h1` with `h2` because each html page should contain only one h1 tag and to follow the [heading hierarchy](https://www.w3.org/WAI/test-evaluate/preliminary/#headings) :blush:

HAMMAS-SALEEM commented on index.html:
```diff
@@ -14,41 +14,251 @@
+        </section>
+        <section class="work-container">
+            <article class="work-item">
+                <header>
+                    <figure class="work-img"><img src="assets/images/project-tonic.png"
+                            alt="Project Tonic Prevew Image"></figure>
```
- [x] Kindly make your card images responsive by giving those images `width: 100%;` :blush: so then the images would be responsive too :+1:

HAMMAS-SALEEM reviewed (changes requested):
# Required Changes :recycle:
![68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f574770523531743971623739534e464f74732f67697068792e676966](https://user-images.githubusercontent.com/93039359/190567385-8e7bb3c0-7d3b-4d25-8385-61d173233786.gif)

## Hey @lmunozm1702 :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Descriptive PR Title

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

lmunozm1702 committed:
Code Review: Just one H1 tag

Update index.html and styles.css to let just one <h1> tag with the main title and another  changes to <h2>

lmunozm1702 committed:
Code Review: card <img> width to 100%

Update style.css to set the card's image width to 100% and get responsiveness.

lmunozm1702 committed:
Code Review: delete max-width property in <body>

Update styless.css to delete the max-width:375px seted to made comparations with Figma template.

lmunozm1702 commented:
All changes are done!

sumairq reviewed (approved):
### Hi @lmunozm1702  , 

<img src="https://media4.giphy.com/media/BPJmthQ3YRwD6QqcVD/giphy.gif"/>

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
